ext/standard/tests/versioning/version_compare.phpt

```diff
@@ -17,6 +17,7 @@ test("1", "2");
 test("10", "2");
 test("1.0", "1.1");
 test("1.2", "1.0.1");
+test("1.2(message)", "1.2(message)");
```

Comment:
Doesnt this test miss a `-` between version-no and message?

Reply:
The actual issue is the trailing bracket 
https://3v4l.org/s6r6c

I could add a test for it